### ISSUE TITLE
Instrumenting solidity 0.5.x contracts with calldata keyword.

### ISSUE THREAD

**Issue by Amxx** [sc-forks/solidity-coverage#316]
When running solidity coverage, I am getting an error:

```
There was a problem instrumenting ./coverageEnv/contracts/XXX.sol: SyntaxError: Expected ")", ",", comment, end of line, or whitespace but "_" found. Line: 50, Column: 20
```

Line 50 is of `XXX.sol` is:

```
49.	function myMethod(
50.		string  calldata _firstArg,
51.		bytes32 _secondArg,
```

If I remove the calldata keyword my code will not compile using the classical workflow, and I'd like to avoid having a specific version of my code for running solidity-coverage

**Comment by Amxx** [comment]
I installed solidity-parser-antlr, but my guess is I need to rebuild solidity-coverage and the documentation doesn't explain how to do this

**Comment by sirhill** [comment]
I have the same exact error with the 'payable' keyword. Should I open a separate issue  ?

**Comment by MariusVanDerWijden** [comment]
No, I found a workaround here: https://github.com/sc-forks/solidity-parser/pull/18,
you have to replace the parser.js file in node-modules/solidity-parser-sc/build/ with this https://raw.githubusercontent.com/maxsam4/solidity-parser/solidity-0.5/build/parser.js.

I think it is time to fork these projects away from sc-forks

**Comment by pinkiebell** [comment]
TL;DR: https://github.com/leapdao/solidity-coverage.git#master

FYI, we have the solidity-coverage and -parser now under our leapDAO umbrella.
This includes support for `calldata`, `payable` and inline function declarations as of now.

**Comment by area** [comment]
I am working on a release that supports solidity 0.5.1, based on the leapdao fork @pinkiebell has mentioned in #318 which should fix your issue - I would be interested in any other odd behaviour you stumbled across while trying it out.

**Comment by cgewecke** [comment]
Fixed in `0.6.0`
